tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/socket/SocketBaseTest.java

```diff
@@ -55,6 +54,7 @@ public void start() throws BallerinaTestException {
         String[] args = new String[] { "-e", "certificate.key=" + privateKey, "-e", "public.cert=" + publicCert };
         serverInstance = new BServerInstance(balServer);
         serverInstance.startServer(balFile, "services", args, requiredPorts);
+        executor.execute(mockSocketServer);
```

Comment:
Any reason to do this change? seems like this is not related to XML, string concatenation.